### DISCUSSION TITLE
UWP Version

### DISCUSSION THREAD

**Discussion by hammondreg** [cefsharp/CefSharp#4374] [Ideas]
Are there any plans to allow CefSharp to run within a UWP application?

**Comment by amaitland** [comment]
Short answer is no. You can use `CefSharp` in a `Desktop Bridge` application.

I believe it's a technical limitation of `UWP`. Effectively only `Microsoft` browsers will run in `UWP`.

 See https://magpcss.org/ceforum/viewtopic.php?f=6&t=16135#p38893

**Comment by hammondreg** [comment]
Thank you Alex, most kind of you to reply

Regards

Ray
________________________________
